### DISCUSSION TITLE
How to use GPU for training

### DISCUSSION THREAD

**Discussion by abccc140** [timeseriesAI/tsai#445] [Q&A]
Thank you for providing such a useful library, but when I am training i fount that my GPU is not in us like below
![image](https://user-images.githubusercontent.com/58618186/160052365-6a3497d0-05c1-4293-9531-ec3e1eb522af.png)

but the computer_setup() shows i have GPU for use 
![image](https://user-images.githubusercontent.com/58618186/160052442-45b526d2-d156-4acf-a31d-04056e26b6d7.png)

So, could anyone give me an example that how to use GPU for training.
Thank you very much!!!!!

**Comment by williamsdoug** [comment]
Tsai builds upon fastai, which builds upon PyTorch at a foundational level.  I suspect the problem you are encountering lives at the torch level.

Here'a a pointer to a StackOverflow that I've found helpful in the past:

https://stackoverflow.com/questions/48152674/how-to-check-if-pytorch-is-using-the-gpu

Your system config shows 2 gpus, so perhaps that is the issue. If the Nvidia 2060 is GPU 1, you may need to use .to(1) to place both your model and data on the correct GPU prior to execution.
